### PR TITLE
Updated Cloud Components release notes for 1.0.2 release

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Updated release notes for Magento Cloud Components 1.0.2 release.

## Affected DevDocs pages

https://devdocs.magento.com/cloud/release-notes/mcc-release-notes.html (new topic)

## Links to Magento source code

[Magento Cloud Components 1.0.2 release content](https://github.com/magento/magento-cloud-components/pulls?q=is%3Apr+is%3Aclosed+label%3A%22Release%3A+1.0.2%22)

whatsnew
Added the [Magento Cloud Components release notes]( https://devdocs.magento.com/cloud/release-notes/mcc-release-notes.html) topic to the _Cloud Guide_.

### CODE REVIEW

File: src/cloud/release-notes/mcc-release-notes.md [hunk 1]
```diff
@@ -15,18 +15,20 @@ The `{{site.data.var.mcc}}` package uses the following version sequence: `<major
 
 This release includes the following updates:
 
--  Item 1
+- {:.new}Extended the functionality of the WARM_UP_PAGES environment variable to support cache preloading for specific product pages. See the [post-deploy variables]({{site.baseurl}}/cloud/env/variables-post-deploy.html#warm_up_pages) topic for a detailed feature description.<!--MAGECLOUD-4444-->
```
Comment (hguthrie) [inline src/cloud/release-notes/mcc-release-notes.md]: In this topic there are several ways the variable name is emphasized, sometimes as inline code, sometimes bold. Does this one need some emphasis?

File: src/cloud/release-notes/mcc-release-notes.md [hunk 2]
```diff
@@ -15,18 +15,20 @@ The `{{site.data.var.mcc}}` package uses the following version sequence: `<major
...
+
+-  {:.fix}Fixed an issue caused by an invalid store URL that causes the post-deploy hook to fail when using the `WARM_UP_PAGES` functionality to populate the cache. This issue occurred only when URL rewrites were disabled.<!-- MAGECLOUD-4094 -->
```
Comment (hguthrie) [inline src/cloud/release-notes/mcc-release-notes.md]: Multiple uses of "cause"... maybe can remove the first one? "Fixed an issue with ..." maybe

Comment (meker12) [inline src/cloud/release-notes/mcc-release-notes.md]:
```suggestion
-  {:.fix}Fixed an issue where an invalid store URL causes the post-deploy hook to fail when using the `WARM_UP_PAGES` functionality to populate the cache. This issue occurred only when URL rewrites were disabled.<!-- MAGECLOUD-4094 -->
```

File: src/cloud/release-notes/mcc-release-notes.md (hunk 1 above)
Comment (meker12) [inline src/cloud/release-notes/mcc-release-notes.md]:
```suggestion
- {:.new}Extended the functionality of the `WARM_UP_PAGES` environment variable to support cache preloading for specific product pages. See the [post-deploy variables]({{site.baseurl}}/cloud/env/variables-post-deploy.html#warm_up_pages) topic for a detailed feature description.<!--MAGECLOUD-4444-->
```

**Pull request merged**